test/data/test_msg_same_id.edn

```diff
@@ -0,0 +1,8 @@
+; Like test_msg.edn but with a different project (same message-id, so equal by misc/message-ids-equal?)
```

Comment:
When would PTC get two messages with the same `message-id` but different contents? 🤔

Reply:
I can't imagine that ever happening, I was just showing the behavior of the equality check (in that this is expected and not a bug--two different messages with the same ID is a bug on the JMS side). If the test makes the behavior more confusing than less then I think it's totally fair to remove it.

Reply:
Thanks for clarifying! I think we can leave it in to show the behavior of the equality check like you mentioned.